http/handler/dockerhub/dockerhub_status.go

```diff
@@ -115,14 +115,16 @@ func getDockerHubLimits(httpClient *http.Client, token string) (*dockerhubStatus
 		return nil, errors.New("failed fetching dockerhub limits")
 	}
 
+	// don't return an error if these headers are missing.  Dockerhub Pro accounts
+	// do not have these headers.  Returning an error adds confusing debug into the browser window.
 	rateLimit, err := parseNumericHeader(resp.Header, "RateLimit-Limit")
 	if err != nil {
-		return nil, fmt.Errorf("Failed fetching RateLimit-Limit header: %w", err)
+		return nil, nil
 	}
 
 	rateLimitRemaining, err := parseNumericHeader(resp.Header, "RateLimit-Remaining")
 	if err != nil {
-		return nil, fmt.Errorf("Failed fetching RateLimit-Remaining header: %w", err)
+		return nil, nil
```

Comment:
with these changes, dockerhub status call will return a json response `200 OK  null` (if I'm not mistaken)
Is that an expected behaviour?
Not a tricky question, genuinely curious.

Reply:
Just a way to return no data and supress the error as it's not really an error.
It was either that or return "{ }".  I went for the nil approach.